api/client/login.go

```diff
@@ -0,0 +1,118 @@
+package client
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+
+	"github.com/appcelerator/amp/api/rpc/oauth"
+	"github.com/fatih/color"
+	"github.com/howeyc/gopass"
+	"golang.org/x/net/context"
+)
+
+func getUsername() (username string) {
+	color.Set(color.FgMagenta, color.Bold)
```

Comment:
@generalhenry We were chatting about this yesterday when I mentioned we need to fix logs. It wasn't my intention that `api/client` have any CLI/terminal concerns. I figured those would be helpers that would be part of `cmd/amp` (we discussed having `cmd/amp/cli` for all the helpers or even just `cli` as I had originally. The `api/client` package would be where I would expect to do things like ensure the security token is included in the grpc context to the backend, etc.